camera_system/camera_calibrator.py

```diff
@@ -0,0 +1,105 @@
+"""カメラキャリブレーションモジュール.
+
+カメラキャリブレーションを行う
+@author kawanoichi
+"""
+import cv2
+from typing import List
+from color_changer import Color
+from color_changer import ColorChanger
+from camera_coordinate_calibrator import CameraCoordinateCalibrator
+from game_info import GameInfo
+
+
+class CameraCalibration:
+    """ゲームエリア認識クラス.
+
+    Attributes:
+        MODE_AREA_XSIZE: 最頻値を求める範囲のxサイズ(奇数)
+        MODE_AREA_YSIZE: 最頻値を求める範囲のyサイズ(奇数)
+    """
+
+    # テンプレートサイズ(奇数)
+    MODE_AREA_XSIZE = 5
+    MODE_AREA_YSIZE = 5
+
+    def __init__(self, read_path):
+        """CameraCalibrationのコンストラクタ.
+
+        Args:
+            read_path: コース画像パス
+        """
+        self.__img = cv2.imread(read_path)
+        self.__save_path = "result_" + read_path
+        self.__color_changer = ColorChanger()
+        self.__coord = CameraCoordinateCalibrator()
+
+    def camera_calibration_start(self):
+        """カメラキャリブレーションを行う関数."""
+        # GUIから座標取得
+        self.__coord.show_window(self.__img)
+
+    def make_game_info(self):
+        """コース情報作成を行う関数."""
```

Comment:
ゲーム（エリア？）情報なのかコース情報なのかは統一した方がいいと思う 

モデルに合わせるならゲームエリア情報だから，以下のようにすればいいかな
```suggestion
    def make_game_area_info(self):
        """ゲームエリア情報作成を行う関数."""
```
この場合`GameInfo`の方も`GameAreaInfo`にするべきだけど...